pkg/ebpf/dns_cache.go

```diff
@@ -18,11 +18,11 @@ type reverseDNSCache struct {
 	size int
 
 	// Telemetry
-	length   int64
-	lookups  int64
-	resolved int64
-	added    int64
-	expired  int64
+	length   int32
+	lookups  int32
+	resolved int32
+	added    int32
+	expired  int32
```

Comment:
I believe the same thing can be accomplished simply by having them first in the structure definition.  Then they can remain int64s